auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -470,17 +345,17 @@ export default defineComponent({
       RESUBMITTED: 'Resubmitted',
       REJECTED: 'Rejected',
       APPROVED: 'Approved',
-      ABANDONED: 'Abandoned'
+      ABANDONED: 'Abandoned',
+      CANCELLED: 'Cancelled'
```

Comment:
@vysakh-menon-aot I know we haven't finished designing this, but this is one way we could show authorizations that a user cancelled (or deleted).

This is an easy change so can we leave it here in case we need it? Can easily remove later if not.